interview-questions.md

```diff
@@ -8,36 +8,97 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer:i beleive that i can go in to my files and Cohort.rb file and type in has_many student and the go into the Students.rb and enter belongs to Cohorts
+
+
+Researched answer: belongs_to establishes a connection with another model . and each new model that this is assoscationed  with will have that forieng key attached .
```

Comment:
It will have a column for foreign keys if we correctly create that model. If not, we would create a migration and add that column.